lib/bemxjst/index.js

```diff
@@ -50,6 +50,10 @@ BEMXJST.prototype.compile = function(code) {
   var self = this;
 
   function applyCtx() {
+    return self.run(self.context.ctx);
```

Comment:
@zxqfox see in context of #203 

`run()` — create new BEMContext
`_run()` — doesnt